source/_components/light.everlights.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "EverLights Light"
+description: "Instructions on how to setup EverLights within Home Assistant."
+date: 2018-12-29
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+featured: false
+ha_release: 0.86
+---
```

Comment:
Is there no logo for EverLights?

Reply:
They do have one. I'll check with them if it can be used here and if so which version of it they want to be used.